weave-js/src/common/css/Base.less

```diff
@@ -1046,9 +1046,12 @@ img.ui.avatar.image {
     .empty-watermark img, .editable-image img, img.image-icon, img.disable-night-mode-filter-revert, .search-nav * {
         filter: none;
     }
-    :not(.night-aware):not(.night-aware-exclude-children) {
+    :not(.night-aware):not(.night-aware-exclude-children):not(.allow-box-shadow) {
         box-shadow: none !important;
     }
+    .allow-box-shadow {
+        box-shadow: 0 12px 24px rgba(0, 0, 0, 0.16)  !important;
+    }
```

Comment:
Need this to allow box shadow for elements that aren't night-aware. Otherwise we have to override all of the night mode auto-inverted styles just to get the box shadow to show up